src/QMCWaveFunctions/IonOrbital.cpp

```diff
@@ -155,7 +156,12 @@ IonOrbital::ratioGrad(ParticleSet& P, int iat, GradType& grad_iat)
 }
 
 void
-IonOrbital::restore(int iat) {}
+IonOrbital::restore(int iat)
```

Comment:
Restore should not be left empty. evaluateLog should directly assign zero.

Reply:
Yes, that was the problem. Please see "fix ratio test". I accidentally left out this piece of the fix when I made this PR. Sorry about that.

Reply:
I saw "fix ratio test". I mean revert the change of restore and assign zero directly in evaluateLog. One goal is to remove all the restore at least in CPU code.

Reply:
Oh I see. I was not aware of that goal. Will clean now